### ISSUE TITLE
More incorrect indentation 

### ISSUE THREAD

**Issue by yha** [julia-vscode/julia-vscode#1946]
```
f(a, b,
    c, d) # Hitting Enter here...
    # <= indents to here

# Pasting here automatically indents the pasted code to align with `c` above
```
The same problem also occurs with comprehensions or array literals on multiple lines.

**Comment by pfitzseb** [comment]
We can't really do anything about this, afaict. Or at least not easily. By indenting the second line manually you basically tell VSCode that that's the new indentation level, and a closing paren shouldn't change that.

**Comment by yha** [comment]
For reference I checked what happens in the C extension, and it also has the first part of this issue: hitting enter after the closing parens follows the manual indentation.
But it doesn't re-indent on paste (although it does have some automatic indentation, at least after braces), which is the more annoying part. So apparently it's possible.
I find I can workaround this problem by setting the Auto Indent setting to "advanced" (instead of the default "full"). But that also turns off useful auto-indentations.

**Comment by yha** [comment]
There's also this extension, which solves the first part of the issue for python, and does some more indentation tricks: https://github.com/kbrose/vsc-python-indent

**Comment by pfitzseb** [comment]
Yeah it's for sure possible to do this, but also fairly annoying implementation-wise. I think the best solution is to get JuliaFormatter in and allow people to perform nice after-the-fact formatting.

**Comment by yha** [comment]
After the fact automatic formatting could be nice, but I don't think it's relevant to this issue at all, honestly. When vscode automatically indents my code wrong, I immediately fix it manually. Doing this with a separate automatic formatter rather than hitting backspace or shift+tab wouldn't be much of an improvement.

Anyway, I found an odd partial workaround: turning the "Format on paste" option *on* causes pasted code *not* to indent.
